README.md

```diff
@@ -80,65 +83,77 @@ Cloud Pak for Data uses the concept of `Deployment Spaces` to configure and mana
+* Click on the top tile for 'Create an empty space':
+
+![Create empty deployment space](doc/source/images/createEmptyDeploymentSpace.png)
+
+* Give your deployment space a unique name, an optional description, then click `Create`.
+
+![Create New deployment space](doc/source/images/createNewDeploymentSpace.png)
+
+You will use this space later when you deploy a machine learning model.
+
+### Add collaborator
 
-#### Add collaborator
 Next, we will add a collaborator to the new deployment space, so that assets we deploy can be monitored in step 4.
 
 * Click on your new deployment space.
 
 ![Select deployment space](doc/source/images/selectNewDeploymentSpace.png)
 
-* Click on the `Access control` tab and then click on `Add collaborators` on the right.
+* Click on the `Access control` tab and then click on `Add collaborators +` on the right.
 
 ![Deployment space access control](doc/source/images/deploymentSpaceAccessControl.png)
 
-* Enter "admin" as a Collaborator and select the user from the drop down list. Then click on the `Add to list` button.
+* Enter "admin" as a Collaborator and select the user from the drop down list. Then click on the `Add to list +` button. Then click the `Add` button to finish adding the collaborator.
 
-> **NOTE:** We are adding the user that configured the machine learning instance for OpenScale monitoring. In this case, the user is the admin user. If the user is someone other than Admin, then that user should be added as a collaborator here
+> **NOTE:** We are adding the user that configured the machine learning instance for OpenScale monitoring. In this case, the user is the admin user. If the user is someone other than "admin", then that user should be added as a collaborator here.
 
-![Deployment space collaborators](doc/source/images/deploymentSpaceAddCollaborator.png)
+![Deployment space add collaborator](doc/source/images/deploymentSpaceAddCollaborator.png)
 
-* Click the `Add` button to finish adding the collaborator. You should be brought back to the deployment space page and see your user ID along with the `Admin` user as collaborators for this space.
+You should be brought back to the deployment space page and see your user ID along with the `admin` user as collaborators for this space.
 
+![Deployment space collaborators](doc/source/images/deploymentSpaceCollaborators.png)
 
 ## 3. Configure OpenScale in a Jupyter Notebook
 
 For this part of the pattern we're going to configure our Watson OpenScale service by running a Jupyter Notebook.
 
 ### Import the notebook
 
-At the project overview, either click the *New Asset* button [1], and choose *Add notebook* or go to the *Notebooks* tab [2] and choose *+ Add notebook* [3].
+At the project overview click the Add to project button, and choose Notebook or click New notebook option next to the Notebooks section.
```

Comment:
This line no longer adds emphasis to the buttons names (or other screen elements to look for) with backticks (or italics or bold -- our style varies even in this one doc).  I.e. `Add to project`, `Notebook`, `New`, `Notebooks` and also the following line has `From URL`.